src/main/java/nl/hsac/fitnesse/fixture/slim/XmlFixture.java

```diff
@@ -90,6 +90,15 @@ public String createFileFromBase64ContentOf(String baseName, String xPath) {
         }
     }
 
+    /**
+     * Validat the loaded xml against a schema in file xsdFileName
+     * @param xsdFileName filename of the xsd to use
+     * @return true if the xml validates against the schema. Throws a descriptive exception otherwise
+     */
+    public boolean validateAgainst(String xsdFileName) {
```

Comment:
Does it also make sense to provide a method where the xsd content is provided by the wiki (preformatted)?